### ISSUE TITLE
Command name "queue:work" or `false` defaults to Laravel native Worker.

### ISSUE THREAD

**Issue by diggersworld** [digbang/safe-queue#8]
I'm attempting to use **0.4.*** on a Laravel 6 project. 

I'm finding that when I set the `command_name`  to either `queue:work` or `false` the `runJob` method in the package Worker never gets called and instead the job is processed by the native Laravel Worker.

If I change the `command_name` to `doctrine:queue:work` or anything other than `queue:work` or `false` and call it, then the `runJob` method in the package Worker is called.

I have done the following:

1. `composer require digbang/safe-queue:0.4.*`
2. Added `Digbang\SafeQueue\DoctrineQueueProvider::class` to the `app.php` file.
3. Create the `safequeue.php` config file and specified the `command_name`.

Have I missed something?

**Comment by bolivir** [comment]
Did you find a solution for this? Im having the same problem on a Laravel 8 project.

**Comment by diggersworld** [comment]
No, so instead, I implemented a helper for closing any open doctrine connections and just called it wherever I needed to use it.

```php
<?php
use Doctrine\Common\Persistence\ManagerRegistry;

class DoctrineCacheBuster
{
    private $managerRegistry;

    public function __construct(ManagerRegistry $managerRegistry)
    {
        $this->managerRegistry = $managerRegistry;
    }

    public function closeEntityManagerConnections() : void
    {
        $connections = $this->managerRegistry->getConnections();
        foreach ($connections as $connection) {
            $connection->close();
        }
    }
}
